### ISSUE TITLE
Superstream no video links found

### ISSUE THREAD

**Issue by ghost** [Stormunblessed/eng-ext#1]
Playing Movie or TV shows of 2023 returning statement "no video links found"

**Comment by hvroun1** [comment]
yes I also have the same problem with superstream on both aniyomi and cloudstream, have you found any workarounds?

**Comment by ghost** [comment]
i think the admin stopped uploading video file to the server
